### DISCUSSION TITLE
Some Relationship methods not available.

### DISCUSSION THREAD

**Discussion by popovicieduard** [adonisjs/core#1425] [Help] [answered]
Noticed that some relationship methods such as fetchOrCreateMany are missing:

```
    const relationshipModels = await model
      .related('relationshipModel')
      .fetchOrCreateMany(key, {params})
```

Are these in development or should I look for a workaround ?

**Comment by thetutlage** [comment, marked as answer]
Yup, needs to be added to the ORM. Missing for now. Added to roadmap https://trello.com/c/5v9uKuMN/33-add-fetchorcreatemany-and-createorupdatemany-to-relationships
